ssc/cmod_pvwattsv1.cpp

```diff
@@ -302,7 +302,7 @@ class cm_pvwattsv1 : public compute_module
 			p_sunup[i] = (ssc_number_t)sunup;
 
 			p_shad_beam[i] = 1.0f;
-			if ( shad.fbeam( i, solalt, solazi ) )
```

Comment:
This had to be reverted before as all tests were failing with row translation calculations in the shading_factors classes. The results of the test were close but off and the old method had to be used.

Reply:
This is different logic than was used before, and all tests pass with this logic.